jdk-recent-unit-tests/src/test/java/com/uber/nullaway/jdk17/NullAwaySwitchTests.java

```diff
@@ -174,4 +175,30 @@ public void testSwitchExprLambda() {
             "}")
         .doTest();
   }
+
+  @Ignore
```

Comment:
Please add a comment indicating we are ignoring because of a crash in the dataflow library